.travis.yml

```diff
@@ -15,6 +15,8 @@ addons:
 before_install:
   - cat test-requirements.txt >> requirements.txt
   - npm install stylelint@7 -g
+  - pip install nltk
+  - python3 -m nltk.downloader punkt averaged_perceptron_tagger maxent_ne_chunker words wordnet
```

Comment:
This line is too long. Run coala locally that will help you find other mistakes. [Traivs](https://travis-ci.org/manu-chroma/username-availability-checker/jobs/454067935#L1082) failed here too.

Reply:
okk